src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--position calendar--month-length calendar-start-day--wed">
+      <span class="day day--1"></span>
+      <span class="day day--2"></span>
```

Comment:
```suggestion
      <span class="calendar__day"></span>
      <span class="calendar__day"></span>
```

do not add unique classes to each cell for this task

also, it should be bem elements of calendar

fix all cases

Reply:
check this checklist point 
<img width="839" alt="image" src="https://user-images.githubusercontent.com/102411130/236753763-108fe579-4778-4a3d-a9a2-702d5228a03b.png">